src/scss/fonts.scss

```diff
@@ -1,19 +1,25 @@
 @font-face {
+  font-display: swap;
```

Comment:
Stavo pensando che forse i `fonts` sono in realtà la nostra `foundations/typography`. Che ne pensi? 

Poi qui farei un mixin privato:
```scss
@mixin _fontFace() {
  font-display: swap;
  font-style: normal;
}

@font-face {
  @include _fontFace();
  font-family: ...
  src: ...
}
```

Addirittura meglio sarebbe: 
- avere una lista di font-families
- ciclare su di esse generando un @font-face per ognuna

Reply:
Si assolutamente, questo file c'è dall'inizio per test, ma tutte queste modifiche le farei nella [PR](https://prima-assicurazioni-spa.myjetbrains.com/youtrack/agiles/95-11/96-1759?issue=PRIMA-11317) della tipografia

Reply:
In questo caso era solo un "ripescare stili veecchi", potremmo in effetti usare `fallback` o `optional`, entrambi però, anche se per poco tempo bloccano comunque il contenuto per poi fare lo "swap" della font. Ne riparliamo comunque nella prossima review per la tipografia ;)